lib/fastlane/plugin/revenuecat_internal/actions/trigger_action_in_circle_ci_action.rb

```diff
@@ -10,14 +10,18 @@ module Fastlane
   module Actions
     class TriggerActionInCircleCiAction < Action
       def self.run(params)
+        action = params[:action]
```

Comment:
I was just thinking that it could be nice to accept random parameters for this... So for example we can run jobs that require passing a version number or things like that... But this is ok for now!

Reply:
Oh so we could add parameters to the CircleCI jobs that will eventually get passed to Fastlane?  I like it